lib/AST/DiagnosticEngine.cpp

```diff
@@ -334,6 +334,12 @@ InFlightDiagnostic::warnUntilSwiftVersion(unsigned majorVersion) {
       .wrapIn(diag::error_in_future_swift_version, majorVersion);
   }
 
+  if (majorVersion == 6) {
```

Comment:
I am so used to `warnUntilSwiftVersion(6)` that I no longer perceive `6` as an argument :)

Reply:
Once I realized I forgot this I checked - we do use `.warnUntilSwiftVersion(5)` in 1 place 🙂